src/components/pollutedLocationCard/PollutedLocationCard.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { BsTrashFill } from "react-icons/bs";
+import { BsTrash } from "react-icons/bs";
+import { RiLogoutCircleRFill } from "react-icons/ri";
+import PollutedLocationDTO from "../../types/backEnd/PollutedLocationDTO";
+
+type PollutedLocation = PollutedLocationDTO & {
+  title?: string;
+};
+
+const PollutedLocationCard: React.FC<PollutedLocation> = ({
+  title,
+  radius,
+  progress,
+}) => {
+  return (
+    <div className="mx-auto bg-white rounded-xl shadow-lg flex items-center space-x-4 sm:w-1/4 md:w-4/5 lg:w-1/4 px-2 pb-2 pt-2">
+      <div className="flex flex-col w-full">
+        <div className="flex flex-row space-x-4">
+          <div className="w-1/2 font-bold">
+            <h1>{title || "Title is missing"}</h1>
```

Comment:
`h2` instead of `h1`. `h2` signals a subheading, which is more like what we're trying to achieve (we'll have many cards), `h1` on the other hand should be the main heading in the page